backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/SystemTime_StepDef.java

```diff
@@ -32,4 +33,10 @@ public void metasfresh_has_the_local_time(String zonedDateTime)
 	{
 		SystemTime.setFixedTimeSource(zonedDateTime);
 	}
+
+	@And("we wait for {string} ms")
```

Comment:
AFAIU this can be {int}. then you don't need the parse

Reply:
Sure, will change